src/app/(layout-with-navigation)/(simulation)/infos/commencer/page.tsx

```diff
@@ -95,6 +95,19 @@ export default function Commencer() {
     PreventNavigationContext
   )
 
+  const currentSimulationRef = useRef(currentSimulation)
+  const [shouldGoToSimulateurPage, setShouldGoToSimulateurPage] =
+    useState(false)
+  useEffect(() => {
+    if (!shouldGoToSimulateurPage) {
+      return
+    }
+    if (currentSimulationRef.current?.polls !== currentSimulation?.polls) {
```

Comment:
Ce sera toujours vrai, tu voulais pas plutôt faire :

```suggestion
    if (currentSimulationRef.current?.polls?.length !== currentSimulation?.polls?.length) {
```

Reply:
Pourquoi ce serait toujours vrai ?

Reply:
Parce que deux arrays ne sont jamais égales sauf si tu compares les mêmes refs. Enfin je crois non ?

Reply:
Yep, my bad. De toute façon ça ne fonctionne pas si la simulation en cours est déjà associée au poll. J'ai modifié